libsolidity/analysis/ControlFlowBuilder.cpp

```diff
@@ -277,6 +317,55 @@ bool ControlFlowBuilder::visit(FunctionCall const& _functionCall)
 				m_currentNode = nextNode;
 				return false;
 			}
+			case FunctionType::Kind::Internal:
+			case FunctionType::Kind::DelegateCall:
+			{
+				solAssert(!!m_revertNode, "");
+				visitNode(_functionCall);
+				_functionCall.expression().accept(*this);
+				ASTNode::listAccept(_functionCall.arguments(), *this);
+
+				auto const* functionType = dynamic_cast<FunctionType const*>(
+					_functionCall.expression().annotation().type);
+
+				if (!functionType->hasDeclaration())
+					return false;
+
+				auto const* unresolvedFunctionDefinition =
+					dynamic_cast<FunctionDefinition const*>(&functionType->declaration());
+
+				auto const& functionDefinition = m_contract ?
+					unresolvedFunctionDefinition->resolveVirtual(*m_contract) :
+					*unresolvedFunctionDefinition;
+
+				if (!functionDefinition.isImplemented())
+					return false;
+
+				if (m_visitedFunctions.count(&functionDefinition))
```

Comment:
Is this parameterized on `m_contract`?

Reply:
Nope. Another thing I forgot: Extracting the right contract instance when `expression()` is a `MemberAccess`. fixing.